bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -217,8 +217,8 @@ protected void initModel() {
                     smartImages[index] = baseResourcePath + DOT +
                         selector + DOT + jpegQuality + DOT + width + DOT + extension +
                         (inTemplate ? Text.escapePath(templateRelativePath) : "") +
-                        (lastModifiedDate > 0 ? "/" + lastModifiedDate +
-                        (StringUtils.isNotBlank(imageName) ? "/" + imageName : "") + DOT + extension : "");
+                        (lastModifiedDate > 0 ? ("/" + lastModifiedDate +
+                        (StringUtils.isNotBlank(imageName) ? ("/" + imageName) : "") + DOT + extension ): "");
```

Comment:
I'd remove the space after "(" and before ")".
This applies to all the changes.